app/sagas/room.js

```diff
@@ -167,7 +182,9 @@ const handleForwardRoom = function* handleForwardRoom({ transferData }) {
 };
 
 const root = function* root() {
-	yield takeLatest(types.ROOM.USER_TYPING, watchUserTyping);
+	yield takeLatest(types.ROOM.USER_TYPING, watchUserActivity);
+	yield takeLatest(types.ROOM.USER_UPLOADING, watchUserActivity);
```

Comment:
If the user is uploading a file on a room and starts uploading on another, it's going to dispatch `types.ROOM.USER_UPLOADING`, which is going to trigger `takeLatest` here.
`takeLatest` is going to take the latest action and discard/cancel the existing ones.

So we can only show and keep showing `uploading` indicator to one room at time.